src/components/Products/MobileLibraries/index.js

```diff
@@ -0,0 +1,137 @@
+import CloudinaryImage from 'components/CloudinaryImage'
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import Link from 'components/Link'
+import { IconEllipsis } from '@posthog/icons'
+import { CallToAction } from 'components/CallToAction'
+
+const librariesData = {
+    web: {
+        name: 'Web',
+        description: 'Use autocapture to collect client-side events without any special instrumentation.',
+        items: [
+            {
+                name: 'JavaScript',
+                url: '/docs/libraries/js',
+                icon: (
+                    <CloudinaryImage
+                        src="https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/integrate/js.svg"
+                        alt="JavaScript"
+                    />
+                ),
+            },
+            {
+                name: 'Next.js',
+                url: '/docs/libraries/next-js',
+                icon: (
+                    <CloudinaryImage
+                        src="https://res.cloudinary.com/dmukukwp6/image/upload/next_js_fa0f2983d8.svg"
+                        alt="Next.js"
+                    />
+                ),
+            },
+            {
+                name: 'React',
+                url: '/docs/libraries/react',
+                icon: (
+                    <CloudinaryImage
+                        src="https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/integrate/react.svg"
+                        alt="React"
+                    />
+                ),
+            },
+        ],
+    },
+    mobile: {
+        name: 'Mobile',
+        description: 'Send custom events and identify users. Record mobile sessions and use feature flags.',
```

Comment:
Missing Flutter here now since the desc. Is more generic and not only about replay